R/find_nldi.R

```diff
@@ -0,0 +1,79 @@
+#' @title 	Retrieve features from the \href{https://labs.waterdata.usgs.gov/api/nldi/swagger-ui/index.html?configUrl=/api/nldi/v3/api-docs/swagger-config}{NLDI}
+#' @description Provides a formal query to the
+#' \href{https://labs.waterdata.usgs.gov/about-nldi/index.html}{Network Linked Data Index}.
+#' The function is useful for topology and location based feature discovery.
+#' A user must supply a starting feature, and can add optional navigation direction(s),
+#' and features to identify on the navigated network.
+#' Valid starting options can be given by one of the following arguments: comid, nwis, huc12,
+#'  wqp, location, and origin.
+#' @param comid an NHDPlusV2 COMID
```

Comment:
Please add expected type to each `@param`

Reply:
This is tough because (excluding leading 0s) can be numeric or character.